guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -15,4 +15,6 @@ There are no upgrade warnings with Foreman {ProjectVersion}.
 [id="foreman-deprecations"]
 == Deprecations
 
-There are no deprecations with Foreman {ProjectVersion}.
+=== Compute resources - oVirt
```

Comment:
FYI: There's an attribute for oVirt: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L107 I am OK as is; I am also OK using oVirt in the heading and then explaining that it affects oVirt itself and any downstream products such as RHV and OLVM.